Source/Core/EllipsoidTangentPlane.js

```diff
@@ -117,7 +132,7 @@ define([
     var projectPointOntoPlaneCartesian3 = new Cartesian3();
 
     /**
-     * Computes the projection of the provided 3D position onto the 2D plane.
+     * Computes the projection of the provided 3D position onto the 2D plane, radially outward from the global origin.
```

Comment:
"... radially outward from the {@link EllipsoidTangentPlane.ellipsoid} coordinate system origin."?